src/main/java/com/puppycrawl/tools/checkstyle/meta/JavadocMetadataScraper.java

```diff
@@ -543,18 +536,9 @@ private static int getParentIndexOf(DetailNode node) {
      * @return parent text
      */
     private static String getParentText(DetailNode nodeParagraph) {
-        String result = null;
-        final Optional<DetailNode> resultNode = getFirstChildOfType(nodeParagraph,
-                JavadocTokenTypes.TEXT, 0);
-        if (resultNode.isPresent()
-                && PARENT_TAG.matcher(resultNode.get().getText()).matches()) {
```

Comment:
this check is not needed since there is same check before invocation of this method.